pm/process_pm.f90

```diff
@@ -100,6 +100,13 @@ program pmbufr
 
     str=adjustl(sta_id(j))
     c_sid=str(2:9)
+    read(c_sid,'(z8)',iostat=ios) z_sid
+    if (ios/=0) then 
+      c_sid="9999999"
+      read(c_sid,'(z8)',iostat=ios) z_sid
```

Comment:
I don't know the details but have a few questions:
1. c_id is declared as char(8) and read using "z8", however, it is only assigned a string with a length of 7.
2. Do we want z_sid to be 99999999 or c_sid to be 99999999?

Reply:
update c_sid=99999999, we need c_sid and write it to BUFR file. z_sid only test if c_sid can be read by Z8 format.